features/torque2/client/job-features.pan

```diff
@@ -38,28 +22,82 @@ function torque_size_to_gb = {
     };
     if (match(ARGV[0], 'tb')) {
         size = split('tb', ARGV[0]);
-        return(to_long(size[0]) * 1000);
+        return(to_long(size[0]) * 1099511627776);
```

Comment:
The pan/units are also a little odd: 1 * MB equals 1

Maybe some more realistic units could be added? somethig like:
```
final variable KiB = 1024;
final variable MiB = 1024 * KiB;
final variable GiB = 1024 * MiB;
final variable TiB = 1024 * GiB;
final variable PiB = 1024 * TiB;
final variable EiB = 1024 * PiB;
final variable ZiB = 1024 * EiB;
final variable YiB = 1024 * ZiB;
```